dev/html/ledger.html

```diff
@@ -170,6 +170,7 @@ <h2 id="transactions">Transactions</h2>
 <li><p>The <em>sender</em> <span class="math inline">\(I\)</span>, which is an address which identifies the account that authorized the transaction.</p></li>
 <li><p>The <em>fee</em> <span class="math inline">\(f\)</span>, which is a 64-bit integer that specifies the processing fee the sender pays to execute the transaction.</p></li>
 <li><p>The first round <span class="math inline">\(r_1\)</span> and last round <span class="math inline">\(r_2\)</span> for which the transaction may be executed.</p></li>
+<li><p>The <em>lock</em> <span class="math inline">\(x\)</span>, which is an optional 32-bit integer specifying mutual exclusion . If <span class="math inline">\(x \neq 0\)</span> (i.e., <span class="math inline">\(x\)</span> is set) and this transaction is confirmed, then this transaction prevents another transaction with the lock set to the same value from being confirmed until <span class="math inline">\(r_2\)</span> is confirmed.</p></li>
```

Comment:
Why just 32 bits?  I think we have no other 32-bit fields.

Reply:
I picked 32 bits because in the worst case, <10M transactions have been confirmed in the last MaxTxnLife blocks, which is bigger than a 16-bit uint and less than a 32-bit uint. No other particular reason.

Is there an advantage of also making it 64-bit?

Reply:
I was mostly thinking it may be easier for applications to use this field if they can dump their own internal ID of the transaction as the lock value.  And those internal IDs might be 64-bit long.

Reply:
In that case, is there not a reason to make it larger; for example, 256 bits (hash size)? I guess 64 bits is probably a common ID value in a way that 256 bits is not. But 32-bit IDs must also be pretty common. I don't have particular opinions here otherwise--we can change to 64 bits just to be consistent with other fields then.

Reply:
Yeah, some kind of hash would actually be even more convenient.  I don't actually care too strongly, though.